card/mod/01_core/set/all/content.rb

```diff
@@ -34,10 +34,6 @@ def clean_html?
   true
 end
 
-def history?
-  false
-end
```

Comment:
Why did you remove it from the core mod? We call that method in the standard and machine mod. If someone doesn't want to use the history mod he wiil get errors.